vnet/lib/vnet/endpoints/1.0/datapaths.rb

```diff
@@ -1,24 +1,24 @@
 # -*- coding: utf-8 -*-
 
 Vnet::Endpoints::V10::VnetAPI.namespace '/datapaths' do
+
   def self.put_post_shared_params
     param :display_name, :String
-    param :is_connected, :Boolean
-    param :dpid, :String, transform: :hex, format: /^(0x)?[0-9a-f]{0,16}$/i, on_error: proc { |error|
-      if error[:reason] == :format
-        raise E::ArgumentError, "dpid must be a 16 byte hexadecimal number. Got: \"#{error[:value]}\""
-      else
-        vnet_default_on_error(error)
-      end
-    }
-    param :node_id, :String
+    param :enable_ovs_learn_action, :Boolean
```

Comment:
How about setting it to true by default here? I know it does that in the database but maybe it's more clear if it's set as such in the webapi too?

Of course you'd have to be careful not to set a default value for PUT though.